### DISCUSSION TITLE
Windows 11?

### DISCUSSION THREAD

**Discussion by ellhans1** [debauchee/barrier#1309] [Q&A]
Hi All,

With the release of Windows 11 today, has anyone upgraded and tried Barrier? If so, did it still work or will there need to be an updated version of Barrier to accommodate this? 

Cheers

**Comment by kvltst** [comment]
About to try it. Will let you know.

**Comment by shymega** [comment]
I doubt we'll be able to support Windows 11 straightaway. Before anyone downvotes this answer, it's important to understand the limited manpower Barrier has. There's work on various other issues that need addressing (including Wayland), but I'll keep a note on Windows 11. Personally, I'm not too worried about Windows 11, as it might just work:tm: with Barrier...

**Comment by kvltst** [comment]
So far so good. Using v2.3.3 Windows Server + Mac client (Big Sur 11.6)

**Reply by shymega** [reply]
OK, cool, so we can advertise Barrier as Windows 11 compatible then. Did you test from v2.3.3. and Git, and use the installer or EXEs?

**Comment by ellhans1** [comment]
Thanks guys - In that case I'm about to install windows 11 as well, I'll report back

**Comment by oasaph** [comment]
Using 2.3.3 + Linux client (Ubuntu 20.04.3 v2.3.2) and didn't found any problems so far

**Comment by shymega** [comment]
I installed Windows 11 the other day. I don't actually use Barrier on a day-to-day basis, my role is primarily triage and repo management. I'd need to install VS tooling to compile, and I don't have the space right now. However, I'll see if I can adjust Azure Pipelines to build on Windows 11...

**Reply by shymega** [reply]
OK, so Azure Pipelines only has Server 2022 as the latest, and that isn't based on Windows 11. If Microsoft does allow Windows 11 builds on AP, then we can explore adding CI builds for that. However, this is unlikely, and I'm afraid the best I can do is up builds to Windows Server 2022 - which is based in Win10 21H2. 

Therefore, all future Barrier builds have *not* been tested on Windows 11. If you want to be sure Barrier will work on your Windows 11 machine, then you're going to have to build it yourself.

**Comment by JayOfDoge** [comment]
confirmed working on windows 11, little sketch but the mouse speed is fixed in the 2.4.0 update and connected much easier. if your having issues like I did originally turn off your firewall on both devices and try it. If that's what's breaking it then allow "Barriers.exe" not just "Barrier.exe" through the firewall and try.

**Comment by vivaanmathur** [comment]
Works perfectly with Windows 11.

**Comment by B1tgeek** [comment]
I font that a bug in the Windows 11, Barrier keep show  Barrier is starting,

**Reply by pic-asso** [reply]
Me too! Has anyone been able to reproduce it as well? Any solutions? Ty in advance!

**Comment by dmendezlopez** [comment]
Windows 10 as server and windows 11 as client, pointer got stuck on bottom right corner and unable no use it. keyboard working fine so far.

**Reply by SaberhagenMagic** [reply]
was it resolved?

**Comment by jforward5** [comment]
Using Multiple monitors (3), Windows 11 (server), iMac (client)
Doesn't work.  I have the FW setup to allow barrier.exe, barriers.exe.  Both client and server start, but the mouse and keyboard won't move to the client.

Works when I use the iMac as the server and the Windows 11 as the client.  Seems like in a multi-monitor situation it doesn't know how to find the edge of the server machine.

**Comment by cretan-ibex** [comment]
Using 2 laptops, 3 monitors , Windows 11 (server), windows 10 (client)

1. Closed(screen down) laptop - Win10(client) 2 external monitors. not using the laptop screen
2. Open (screen open) laptop - Win11(server) 1 external monitor the other screen is the laptop screen itself.

Only one issue observed. 
I will close both computers in hibernation mode at the end of the day. The next morning both computer are started. The client restarts and works correctly. The server will start but the client cannot connect to the server. The server needs to be stopped then started again. Stop/start of server is not closing the barrier app but just clicking on the stop then start buttons after right clicking of barrier app in start menu tray. All works well for the rest of the day.

**Comment by top-5** [comment]
Windows 11 client + Windows 11 server:

- connection established.
- keyboard works.
- mouse clicks work, but mouse pointer gets stuck in the bottom-right corner.

**Reply by SaberhagenMagic** [reply]
I have the same problem!

**Comment by HamzaHajeir** [comment]
I've one machine runs Windows 11 and the other runs Windows 10, unfortunately couldn't connect the two together.
Here's my [discussion topic](https://github.com/debauchee/barrier/discussions/1909).

**Comment by quartzeye** [comment]
I have 2-Windows 10, 1-Fedora 37, and 1-Windows 11.  Server is on one of the Windows 10 machines.  Barrier works seamlessly across all.  One abnormality though.  When the Win11 lock screen and screensaver are up and I mouse over to the Win11, as soon as the screensaver turns off, the cursor jumps to the Win10 server.  I can mouse back over but when I get the login prompt on the Win11 machine it jumps back to the Win10 server again. after I enter the PIN and the desktop appears, the cursor once again jumps to the Win10 server.  After that it works normally but this behavior occurs every time the Win11 client goes to a lock-screen and screensaver.  I can run a mouse-jiggle script to prevent the screensaver activating and I have put an enhancement request to add a toggle-abler mouse-jiggle capability in the client app so one could prevent screen-locking and screensavers when that capability is disabled via group policy or other security means.

**Reply by windoze** [reply]
You can set "Elevate" to "Always" in the "Settings" dialog.
This will make Barrier client work in the UAC/Login screen so you can use it to input PIN/password.
